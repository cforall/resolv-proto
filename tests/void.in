foo-v 1
1 foo-n 1

%%

foo( 1 )
