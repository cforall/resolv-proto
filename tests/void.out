
ERROR: ambiguous resolution for foo( 1 )
